cabby/geo/walk.py

```diff
@@ -288,11 +292,10 @@ def get_pivot_beyond_goal(map: map_structure.Map,
 
     try:
```

Comment:
This is a very long try block. Can you localize it to just the area where an exception might be thrown and capture just that exception? 

I gave the link in another comment, but here is the style guide on exceptions, also relevant here: https://google.github.io/styleguide/pyguide.html#24-exceptions